cuda/test/matrix/csr_kernels.cpp

```diff
@@ -313,6 +319,31 @@ TEST_F(Csr, AdvancedApplyToDenseMatrixIsEquivalentToRefWithMergePath)
     GKO_ASSERT_MTX_NEAR(dresult, expected, 1e-14);
 }
 
```

Comment:
one more newline

Reply:
Thanks! Maybe I can add it directly this way:
```suggestion


```